.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Hi! Thanks for your commit! Could this file be removed? There's no relation with the project I think.